src/main/java/com/google/cloud/tools/appengine/api/devserver/DefaultRunConfiguration.java

```diff
@@ -25,6 +25,7 @@
 public class DefaultRunConfiguration implements RunConfiguration {
 
   private List<File> appYamls;
+  private List<File> serviceDirectories;
```

Comment:
@loosebazooka at the core library level we're always going to have to work in terms of output directories don't you think?